public/app/plugins/datasource/prometheus/querybuilder/binaryScalarOperations.ts

```diff
@@ -77,7 +77,11 @@ export const binaryScalarOperations: QueryBuilderOperationDef[] = binaryScalarDe
   const params: QueryBuilderOperationParamDef[] = [{ name: 'Value', type: 'number' }];
   const defaultParams: any[] = [2];
   if (opDef.comparison) {
-    params.unshift({ name: 'Bool', type: 'boolean' });
+    params.unshift({
+      name: 'Bool',
+      type: 'boolean',
+      description: 'If checked comparison will return 0 or 1 for the value rather than filtering.',
+    });
```

Comment:
would it make sense to move the bool param to be last? feels like it will be more rarely used and is optional, so having it first feels like it taking up the primary space for this operation

Reply:
Yeah that makes more sense

Reply:
Will do it in separate PR though